src/boundary_operators.jl

```diff
@@ -65,6 +65,21 @@ function convolve_BC_left!{T<:Real,S<:SVector,RBC}(x_temp::AbstractVector{T}, x:
 end
 
 
+function convolve_BC_left!{T<:Real,S<:SVector,RBC}(x_temp::AbstractVector{T}, x::AbstractVector{T}, A::LinearOperator{T,S,:None,RBC})
+    Threads.@threads for i in 1 : A.boundary_point_count
+        @inbounds for i in 1 : A.boundary_point_count
+            bc = A.low_boundary_coefs[i]
+            tmp = zero(T)
+            startid = max(0,i-1-div(A.stencil_length, 2))
```

Comment:
It takes into account that we don't use the boundary node in the cases under fornberg.jl:109. The issue here is that A.boundary_point_count is too large for what `BC=:None` needs.